core/feconf.py

```diff
@@ -561,17 +561,22 @@ def get_empty_ratings() -> Dict[str, int]:
 
 # NOTE TO RELEASE COORDINATORS: Replace this project id with the correct oppia
 # project id when switching to the prod server.
-OPPIA_PROJECT_ID = 'dev-project-id'
+OPPIA_DEV_PROJECT_ID = 'dev-project-id'
 GOOGLE_APP_ENGINE_REGION = 'us-central1'
 
-# This flag is used for distinguishing the prod/test environments for feature
-# gating.
-ENV_IS_OPPIA_ORG_PRODUCTION_SERVER = bool(OPPIA_PROJECT_ID == 'oppiaserver')
+# This is used for comparing with project id to distinguis the prod/test
+# environments for feature gating.
+PROD_SERVER_ID = 'oppiaserver'
 
-# NOTE TO RELEASE COORDINATORS: Replace these GCS bucket paths with real prod
-# buckets. It's OK for them to be the same.
-DATAFLOW_TEMP_LOCATION = 'gs://todo/todo'
-DATAFLOW_STAGING_LOCATION = 'gs://todo/todo'
+OPPIA_PROJECT_ID_TO_SITE_URL_MAP = {
```

Comment:
Hm, I think we shouldn't make the changes shown here (lines 571-576), because we should not have any of the per-server configuration within the Oppia codebase itself. This site URL is mainly for emails right? Can we make it a platform param like SITE_URL_PREFIX (with a description that it is used for email functionality) -- but for this one we should introduce the platform params, populate them first, and only then switch the email functionality over to use them?

Reply:
Makes sense, noted in testing doc.